js/components/generate/generate.js

```diff
@@ -1,354 +1,6 @@
 import React from "react"
```

Comment:
Seems like this import is no longer necessary

Reply:
Surprisingly JSX syntax cannot be used without importing React. Surprisingly because I read that this should be possible from React 18 (with JSX transform), so I'm not sure why it still needs it, I can look into it.

***
![Screenshot 2024-06-24 at 12 21 38 AM](https://github.com/Courseography/courseography/assets/129094642/fbb6816b-2e53-4f74-949b-e916d274f65c)
***
![Screenshot 2024-06-24 at 12 21 30 AM](https://github.com/Courseography/courseography/assets/129094642/8c9271a2-749d-4e9e-bc27-bb09c18957dc)
***

Reply:
@mimischly7 probably needs the `.jsx` extension at the end of this filename?

Reply:
I left you a slack on this